pdagent/heartbeat.py

```diff
@@ -63,15 +63,15 @@ def __init__(
         self._system_info = system_info
         # The following variables exist to ease unit testing:
         self._source_address = source_address
-        self._urllib2 = httpswithverify
+        self._urllib2 = http
```

Comment:
Can fix this naming on a subsequent pass since this isn't remotely `urllib2`.

Reply:
lol :sparkles: